### ISSUE TITLE
incompatible with goquery

### ISSUE THREAD

**Issue by markelog** [jarcoal/httpmock#18]
``` go

httpmock.Activate()
defer httpmock.DeactivateAndReset()

httpmock.RegisterResponder("GET", "https://api.mybiz.com/articles.html",
  httpmock.NewStringResponder(200, `<html>...</html>`))


// and

goquery.NewDocument("https://api.mybiz.com/articles.html")
```

Will give `Response.Request is nil`

Since `goquery` looks at the `Response` property which seems to not be populated

**Comment by pocke** [comment]
https://github.com/jarcoal/httpmock/pull/11
Maybe the PR solves your problem.

**Comment by markelog** [comment]
Yeah! That should work, depressing it wasn't landed :/

**Comment by dlebech** [comment]
@pocke @markelog This has now been fixed in #11 which has been added to the v1 branch of this library. Please see the updated README on how to update. Cheers :smiley:
